.woodpecker.yml

```
@@ -19,7 +21,9 @@ steps:
   ansible_lint:
     image: alpine:3
     commands:
-      - apk add ansible ansible-lint
+      - apk add pipx
+      - pipx install --include-deps ansible ansible-lint
+      - export PATH=/root/.local/bin:$${PATH}
```

```suggestion
      - export PATH=/root/.local/bin:$${PATH}
      - apk add pipx
      - pipx install --include-deps ansible
      - pipx inject --include-deps ansible ansible-lint
```

same as above, moving `$PATH` to the start gets rid of the warning during `pipx install` about the bin path not being in `$PATH`, but it doesn't really affect the outcome.

the main change is not installing both `ansible` and `ansible-lint` in the same `pipx` execution.
I haven't used pipx before, but using it this way creates a venv for every package. this results in the `ansible-lint` venv not having the packages that exist in the `ansible` venv. [`pipx inject`](https://pipx.pypa.io/stable/docs/#pipx-inject) seems to be just what we are looking for for this to install `ansible-lint` into the same venv as `ansible`.

the result is a clean woodpecker test: https://wpci.lem.rocks/repos/3/pipeline/19/5